src/cdi_CPEloss/Tabular_CP_Eloss.cc

```diff
@@ -0,0 +1,246 @@
+//----------------------------------*-C++-*----------------------------------//
+/*!                                                                              
+ * \file   cdi_CPEloss/Tabular_CP_Eloss.cc                                     
+ * \author Ben R. Ryan                                                        
+ * \date   2019 Nov 4                                             
+ * \brief  Tabular_CP_Eloss member definitions.                                 
+ * \note   Copyright (C) 2016-2019 Triad National Security, LLC.                 
+ *         All rights reserved. */
+//---------------------------------------------------------------------------//
+
+#include "Tabular_CP_Eloss.hh"
+
+using std::stod;
+using std::stoi;
```

Comment:
Technically this is safe because the namespace injection is local to the current translation unit.  However, I would prefer to see these `using` statements inside function scopes where they are needed.